### PR TITLE
Authentication devise Feature

### PR DESCRIPTION
**Pull request by ghost**
In this project, we built an exclusive clubhouse where your members can write embarrassing posts about non-members.
In this project, we create a simple Post with Authentication feature to learn the basics of Ruby on Rails including

Models, Views, and Controllers (MVC)
Data Structures & Relationships
Routing
Migrations
Views with forms, helpers
RESTful design
Adding gems for extra features
Authentication with 'devise'

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -9,23 +10,21 @@ def index
...
-  def edit
-  end
+  def edit; end
 
   # POST /posts
   # POST /posts.json
   def create
     @post = Post.new(post_params)
-
+    @post.user = current_user
```
Comment (elmejdki) [inline app/controllers/posts_controller.rb:27]: use this instead: `@post = current_user.posts.build(post_params)`

**Review by elmejdki — CHANGES_REQUESTED** [review]
# status: require changes :recycle: 

Great job so far @team :clap: :clap: :clap:, your project almost complete :checkered_flag: you need just to take care of some minor changes :hammer: :wrench:.

## Required Changes:

- [x] The application has Sign in/Sign out links present in any view of the app:
image

- [x] It creates the application routes using `resources :posts, only: [:new, :create, :index]`
image

- [x] You have a broken link in your Readme please fix it:
image

- [x] There should be absolutely NO logic in views. >Views are only for displaying information that became available to it from its corresponding controller not generating it. View-related logic can be encapsulated in View helpers (app/helpers/\*_helper.rb).

### Please make all the required changes, before you submit another code review.
## Happy Coding :tophat: 
_don't hesitate to contact me on slack if you have any type of question._

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUSCHANGES REQUIRED ♻️ 

Hello team! ✋ 
You did a great job implementing the suggested changes by the previous TSE! 👍 
However, we still need to make some extra changes to get this project approved 💪 
Kindly, check my comments below 👇  apply the changes and resubmit your project for a new code review ⏩ 
Good luck! 😃 
## Mandatory changes
- As previous TSE suggested, kindly modify your create action in your post controller, use something like this `@post = current_user.posts.build(post_params)`, it is the most efficient way to assign your post to the user
- Kindly remove all the actions your are not using in your project, ie. destroy, update, etc...
- In your readme file, in the authors section, kindly add your username to the links for every field(LinkedIn, GitHub, Twitter...)

**Review by Temmarie — APPROVED** [review]
## Status: Approved :heavy_check_mark:

### Hi @Igors78 and @CornerstoneII ! 👋 👋 

#### Thank you for making all of the required changes, your project is approved :clap: :clap: :clap: :tada:

#### You may merge, good luck with your next projects! :rocket:
#### Happy Coding! :+1:
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

**Pull request merged**